packages/service-library/tests/test_archiving_utils.py

```diff
@@ -182,6 +186,10 @@ def assert_unarchived_paths(
         for f in src_dir.rglob("*")
         if is_file_or_emptydir(f)
     )
+    if surrogate_replace:
+        expected_tails = {
+            x.encode(errors="replace").decode("utf-8") for x in expected_tails
+        }
     assert got_tails == expected_tails
```

Comment:
This assert is a declaration of intent, right? You should add ``# nosec`` to avoid code-climate to complain

Reply:
not necessary, it will not complain as it is part of a test

Reply:
true ... i did not notice it was in a test. my bad